packages/core/src/components/radio/radio.scss

```diff
@@ -0,0 +1,105 @@
+@use '../../helpers';
+@use '../../mixins';
+
+@mixin Radio() {
+  $boundary: helpers.space(6);
+  $size: helpers.space(2.75);
+  $border-width: 1px;
+
+  .ods-radio-label {
+    @include mixins.interactive();
+
+    $padding: ($boundary - $size) / 2;
+
+    display: inline-flex;
+    flex-direction: row;
+    padding: $padding;
+    position: relative;
+    transition: background-color, border-color 0.15s ease-out;
+
+    &.-bordered {
+      border: $border-width solid helpers.color('border-separator');
+      border-radius: helpers.border-radius('medium');
+      display: flex;
+      padding: $padding - $border-width;
+    }
+
+    &.-bordered.-checked {
+      background-color: helpers.color('background-action-subtle');
+      border-color: helpers.color('border-action-subtle');
+    }
+
+    &.-disabled {
+      border-color: helpers.color('border-disabled');
+    }
+
+    > :not([aria-hidden='true']) {
+      display: flex;
+      flex-direction: column;
+      padding-left: helpers.space(1.625);
+      padding-right: helpers.space(0.375);
```

Comment:
The paddings can be substituted by `gap` on the parent:
```scss
.ods-radio-label {
  gap: helpers.space(1.5);
}
```

The column flex is also not necessary, you can simply render blocks, e.g.
```html
<div>Label</div>
<div>Description</div>
```

Reply:
Thanks, I can use `gap` instead of left padding for sure, however value should be `1.625` because it should be `13px` if I understand correctly:

<img width="137" alt="Screenshot 2021-01-06 at 18 03 32" src="https://user-images.githubusercontent.com/20243687/103790155-bcbfa900-5038-11eb-9e95-d09f03aef86f.png">

Reply:
Yup, you're right. Just used `1.5` for simplicity.
And I guess the right padding can be applied to `label > span` instead of all its children.

It's a bit unintuitive how our spacings in there work, but it's neat that we don't need event listeners and can use the `label`, I like it.